xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -386,7 +386,7 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
     m_pFormatContext->pb = m_ioContext;
 
-    AVDictionary *options = NULL;
+    AVDictionary *options = GetFFMpegOptionsFromInput();
```

Comment:
CDVDDemuxFFmpeg reads data from an inputstream, CDVDInputStreamXYZ. ffmpeg options are set if input stream is DVDInputStreamFFmpeg. 
In the case you linked, the m3u is not opened by DVDInputStreamFFmpeg, otherwise the options would have been set in the block above.

btw: forum posts with truncated logs are in gerneral ignored